src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.searchForm {
+  margin-top: 20px;
+}
+
+.searchPic {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  padding-top: 25px;
+  padding-left: 26px;
+  padding-bottom: 26px;
+}
+
+.searchPicSmall {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  padding-top: 15px;
+  padding-left: 13px;
+  padding-bottom: 16px;
+}
+
+.searchBar {
+  padding: 0;
+  outline: none;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir-Book, sans-serif;
+}
+
+.searchBarBig {
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.searchBarSmall {
+  padding-left: 33px;
+  padding-top: 12px;
+  padding-bottom: 10px;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.searchBar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.searchBar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir-Heavy, sans-serif;
+}
+
+.searchBarBig::placeholder {
+  font-family: Avenir-Book, sans-serif;
+  position: relative;
+  color: #3d4e61;
+}
+
+.searchBarSmall::placeholder {
+  font-family: Avenir-Book, sans-serif;
+  position: relative;
+  color: #3d4e61;
+}
```

Comment:
you repeating the same styles, if you will correct comment above you will use only one class instead of two